infrastructure/quick-lint-js-web/apache2/apache2.conf

```diff
@@ -220,6 +220,14 @@ CustomLog ${APACHE_LOG_DIR}/matomo_analytics.log matomo_analytics
 
 FileETag INode MTime
 
+# FCGI
+AddHandler fcgid-script .fcgi .php .fpl
```

Comment:
Do we need this directive at all? We already have a similar directive in `fcgid.conf`.